iocage/lib/ioc_plugin.py

```diff
@@ -541,7 +539,7 @@ def __fetch_plugin_post_install__(self, conf, _conf, jaildir, uuid):
                 iocage.lib.ioc_common.logit(
                     {
                         "level": "INFO",
-                        "message": "Running post_install.sh"
+                        "message": "\nRunning post_install.sh"
```

Comment:
Why do you have to add a \n to every message? Isn't however is receiving these messages supposed to add the line break?